R/pkg/R/generics.R

```diff
@@ -771,6 +771,10 @@ setGeneric("over", function(x, window) { standardGeneric("over") })
 
 ###################### WindowSpec Methods ##########################
 
+#' @rdname orderBy
+#' @export
```

Comment:
Perhaps to orderBy? When applied to SparkDataFrame, it simply calls the arrange function.